<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Terminal2</class>
 <widget class="QDialog" name="Terminal2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Terminal2</string>
  </property>
  <widget class="QLabel" name="lblwelcome">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblconnect">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblbalance">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblpassword">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblbank">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bank:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtPass">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>97</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnback">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAct">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtBalance">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="bankBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bank1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bank2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbllogin">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>32</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Login :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnadd">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtLogin">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>71</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnwithdraw">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnlogin">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>123</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QPushButton" name="brnLogout">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtLogin</tabstop>
  <tabstop>txtPass</tabstop>
  <tabstop>btnlogin</tabstop>
  <tabstop>btnback</tabstop>
  <tabstop>btnadd</tabstop>
  <tabstop>btnwithdraw</tabstop>
  <tabstop>txtAct</tabstop>
  <tabstop>bankBox</tabstop>
  <tabstop>txtBalance</tabstop>
  <tabstop>brnLogout</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
